README.md

```diff
@@ -9,16 +9,38 @@
 
 ## Overview
 
-HealthyHome is an open-source smart home device which monitors the indoors environment conditions, based on a Raspberry Pi.
+HealthyHome is an open-source project centred around a smart home device which monitors the indoor environment conditions of a home.
+
+### Objectives
+
+-   Allow a user to characterise important parameters of their indoor home environment
+-   Provide easy access to both real-time and historical versions of this information
+-   Allow interested parties to recreate this project
+
+### Features
+
+-   Monitors the ambient temperature, barometric pressure, relative humidity, UV index, ambient light levels and concentration of particles in the air
+-   Calculates the indoor air quality (IAQ) index
+-   Provides both real-time and historical measurements
+-   Communicates data to a client (e.g., a smart phone)
+-   Allows user to access real-time and historical data using an application on a client
+-   Uses GNU General Public License 3 (GPL-3.0) so that all the information on the github repository can be used to recreate the project
+
+## Prototype Hardware
+
+-   [Panasonic SN-GCJA5](https://na.industrial.panasonic.com/products/sensors/air-quality-gas-flow-sensors/lineup/laser-type-pm-sensor/series/123557/model/123559): A particulate matter sensor which measures the concentration of particles in the air
+-   [Bosch Sensortec BME680](https://www.bosch-sensortec.com/products/environmental-sensors/gas-sensors/bme680/): A 4-in-1 sensor which measures ambient temperature, relative humidity, barometric pressure, and volatile organic compounds (VOC). It uses these measurements to calculate the IAQ index.
```

Comment:
The IAQ index is a parameter we calculate from all the relevant sensorial information.

Reply:
Removed IAQ explanation from line 32.